source/fundamentals/collations.txt

```diff
@@ -4,5 +4,302 @@ Collations
+field and specifies a new collation with an ``"en_US"`` locale:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: go
+
+       myCollation := &options.Collation{Locale: "en_US"}
+       opts := options.Index().SetCollation(myCollation)
+
+       indexModel := mongo.IndexModel{
+         Keys:    bson.D{{"name", 1}},
+         Options: opts,
+       }
+
+       name, err := coll.Indexes().CreateOne(context.TODO(), indexModel)
+       if err != nil {
+         panic(err)
+       }
+       fmt.Println("Name of Index Created: " + name)
+
+   .. output::
+      :language: none
+      :visible: false
+
+       Name of Index Created: name_1
+
+
+Set a Collation on an Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Operations that read, update, and delete documents from a collection can use collations. 
+
+The following example uses the ``Find()`` method to return documents with ``length`` 
+values greater than ``"1000"``. The ``NumericOrdering`` collation field has a value of
+``true`` to ensure numeric comparison, rather than string comparison:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: go
+
+      filter := bson.D{{"length", bson.D{{"$gt", "1000"}}}}
+      myCollation := &options.Collation{Locale: "en_US", NumericOrdering: true}
+      opts := options.Find().SetCollation(myCollation)
+
+      cursor, err := coll.Find(context.TODO(), filter, opts)
+      
+      var results []bson.D
+      if err = cursor.All(context.TODO(), &results); err != nil {
+         panic(err)
+      }
+      
+      for _, result := range results {
+         fmt.Println(result)
+      }
+
+   .. output::
+      :language: none
+      :visible: false
+
+      [{name Infinite Jest} {author Wallace} {length 1104}]
+      [{name Ça} {author King} {length 1138}]
+      [{name Les Misérables} {author Hugo} {length 1462}] 
+
+Without specifying a collation with a ``NumericOrdering`` field set to ``true``, the
+same ``Find()`` operation would compare ``length`` values as strings. For example, the
+operation would consider the string ``"824"`` as greater than ``"1000"``. The
+output would resemble the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   [{name Emma} {author Austen} {length 474}]
+   [{name Cryptonomicon} {author Stephenson} {length 918}]
+
+Additional Information
+----------------------
+
+To learn more about the ``Find()`` and ``UpdateOne()`` methods mentioned,
+see the following guides:
+
+- :ref:`golang-retrieve`
+- :ref:`golang-change-document`
+
+To learn more about collations, visit the following manual pages:
+
+- :manual:`Collation </reference/collation/#collation-document>`
+- :manual:`Collation Locales and Default Parameters </reference/collation-locales-defaults/#supported-languages-and-locales>`
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+To learn more about any of the methods discussed in this
```

Comment:
Question: What does "any of" do in this sentence? Is there a reason it was included?

```suggestion
To learn more about the methods discussed in this
```

Reply:
I took out "any of"